### ISSUE TITLE
Don't Have loadURL function

### ISSUE THREAD

**Issue by KimberleeDArmstrong** [phonegap/phonegap-plugin-contentsync#145]
cordova version :  6.3.1
cordova ios platform version: 4.2.1
XCode version: 8.0
iOS Version: 10.1
ContentSync version: 1.2.3
WKWebViewEngine version: 1.1.0

When I look at "ContentSync.m" or go into the javascript console and type "ContentSync."  There is no function loadURL available.  

I am wanting to use loadURL function because document.location = data.localPath + '/index.html' is returning an error "Failed to load resource: The operation couldn't be completed. Operation not permitted

Any ideas?

**Comment by KimberleeDArmstrong** [comment]
OK. So I added the plugin using cordova plugin add phonegap-plugin-contentsync and did not have the loadURL function.  That was my fault.  I added it using cordova plugin add https://github.com/phonegap/phonegap-plugin-contentsync and now I do have the loadURL function.

However the error is the same with the ContentSync.loadUrl call "Failed to load resource: The operation couldn't be completed. Operation not permitted"
